### PR TITLE
setup for creating sharded DB

### PR DESCRIPTION
**Pull request by snopoke**
@czue

### CODE REVIEW

**Comment by czue** [comment]
reviewed pretty quickly but looks good!

**Comment by czue** [comment]
(assume you'll ping when you want merged)

**Comment by snopoke** [comment]
:white_check_mark:

**Comment by snopoke** [comment]
:white_check_mark: for realz

File: ansible/deploy_commcarehq.yml [hunk 1]
```diff
@@ -5,7 +5,7 @@
    - celery
    - proxy
    - pillowtop
-   - postgreql
+   - postgresql
```
Comment (czue) [inline ansible/deploy_commcarehq.yml]: :+1: should have caught this

**Comment by czue** [comment]
:last_quarter_moon_with_face: :first_quarter_moon_with_face:

**Pull request merged**